app/viewmodels/AddressFormattingHelper.scala

```diff
@@ -18,8 +18,20 @@ package viewmodels
 
 import models.backend.UkAddress
 import play.twirl.api.{Html, HtmlFormat}
+import uk.gov.hmrc.govukfrontend.views.Aliases.HtmlContent
 
 object AddressFormattingHelper {
+
+  def formatBusinessAddress(ukAddress: UkAddress, tradingName: Option[String]): HtmlContent = {
+    HtmlContent(tradingName.fold("")(tradingName => tradingName + "<br/>") +
+      ukAddress.lines.map(line => if (line.isEmpty) {
+        ""
+      } else {
+        HtmlFormat.escape(line).toString() + "<br/>"
+      }).mkString +
+      s"""<span class="nowrap" style="white-space: nowrap;">${ukAddress.postCode}</span>""")
```

Comment:
Why do we need the nowrap? I feel like there would be a cleaner way to do this - depending on the exact reasoning why

Reply:
I was following the pattern used in registration, however, as post code is on it's own line in this usage, it should not be needed.  Removed and re-pushed code.

Reply:
updated